base/src/main/java/com/smartdevicelink/proxy/rpc/HMICapabilities.java

```diff
@@ -58,11 +58,17 @@ public boolean isNavigationAvailable(){
 		 }
 		 return (Boolean)available;
 	 }
-	 
+
+	 @Deprecated
 	 public HMICapabilities setNavigationAvilable( Boolean available) {
```

Comment:
Please add javadoc as to what method to use now that this one is deprecated. Since this is a spelling fix to a method name please use the new method inside this method to avoid code redundancy.

Reply:
This should be done for all method name spelling fixes as well. I won't leave comments on each.